spec/system/admin/orders_spec.rb

```diff
@@ -662,17 +662,77 @@ def extract_pdf_content
             end
           end
 
+          shared_examples "prints invoices accordering to column ordering" do
+            it "bulk prints invoices in pdf format" do
+              page.find("span.icon-reorder", text: "ACTIONS").click
+              within ".ofn-drop-down .menu" do
+                expect {
+                  page.find("span", text: "Print Invoices").click # Prints invoices in bulk
+                }.to enqueue_job(BulkInvoiceJob).exactly(:once)
+              end
+
+              expect(page).to have_content "Compiling Invoices"
+              expect(page).to have_content "Please wait until the PDF is ready " \
+                                           "before closing this modal."
+
+              perform_enqueued_jobs(only: BulkInvoiceJob)
+
+              expect(page).to have_content "Bulk Invoice created"
+
+              within ".modal-content" do
+                expect(page).to have_link(class: "button", text: "VIEW FILE",
+                                          href: /invoices/)
+
+                invoice_content = extract_pdf_content
+
+                expect(
+                  invoice_content
+                ).to match(/#{surnames[0]}.*#{surnames[1]}.*#{surnames[2]}.*#{surnames[3]}/m)
```

Comment:
Note that `invoice_content` is an array of 4 strings, one for each invoice. If we join them into one string then the test passes.